### PR TITLE
[#5673] Add test and docs about how to use GCS in Hive

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

1. Release a new docker Hive image to support GCS.
2. Add related test based on the new image.
 
### Why are the changes needed?

For users convenience. 

Fix: #5673 

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

new IT `CatalogHiveGCSIT`

### CODE REVIEW

**Comment by yuqi1129** [comment]
This PR is not ready for review until https://github.com/apache/gravitino/pull/5558 is merged.

**Comment by yuqi1129** [comment]
> This PR is not ready for review until #5558 is merged.

This PR is ready for review.

File: docs/hive-catalog-with-s3-adls-gcs.md [hunk 1]
```diff
@@ -11,14 +11,13 @@ license: "This software is licensed under the Apache License version 2."
 
 Since Hive 2.x, Hive has supported S3 as a storage backend, enabling users to store and manage data in Amazon S3 directly through Hive. Gravitino enhances this capability by supporting the Hive catalog with S3, allowing users to efficiently manage the storage locations of files located in S3. This integration simplifies data operations and enables seamless access to S3 data from Hive queries.
 
-For ADLS (aka. Azure Blob Storage (ABS), or Azure Data Lake Storage (v2)), the integration is similar to S3. The only difference is the configuration properties for ADLS(see below). 
+For ADLS (aka. Azure Blob Storage (ABS), or Azure Data Lake Storage (v2)) and GCS (Google Cloud Storage), the integration is similar to S3. The only difference is the configuration properties for ADLS and GCS (see below).
```
Comment (FANNG1) [inline docs/hive-catalog-with-s3-adls-gcs.md:14]: rename file to `hive-catalog-with-cloud-storage`?

File: docs/hive-catalog-with-s3-adls-gcs.md [hunk 2]
```diff
@@ -225,13 +240,17 @@ To access S3-stored tables using Spark, you need to configure the SparkSession a
             .config("spark.sql.catalog.{hive_catalog_name}.fs.s3a.endpoint", getS3Endpoint)
             .config("spark.sql.catalog.{hive_catalog_name}.fs.s3a.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem")
 
-            ## This two is for Azure Blob Storage(ADLS) only
+            // This two is for Azure Blob Storage(ADLS) only
```
Comment (FANNG1) [inline docs/hive-catalog-with-s3-adls-gcs.md:243]: Azure Blob Storage is not ADLS, YES?

**Comment by FANNG1** [comment]
LGTM except minor comments

File: docs/hive-catalog-with-s3-adls-gcs.md (hunk 1 above)
Comment (yuqi1129) [reply docs/hive-catalog-with-s3-adls-gcs.md:14]: Yes, it's a good idea, I take it.

File: docs/hive-catalog-with-s3-adls-gcs.md (hunk 2 above)
Comment (yuqi1129) [reply docs/hive-catalog-with-s3-adls-gcs.md:243]: Azure Blob Storage refers to the Azure storage concept, including ordinary Azure blob storage, ADLS, and ADSLV2.

File: docs/hive-catalog-with-s3-adls-gcs.md (hunk 1 above)
Comment (yuqi1129) [reply docs/hive-catalog-with-s3-adls-gcs.md]: fix

**Comment by jerryshao** [comment]
I'm OK with the change, @FANNG1 do you want to take another look?

**Review by FANNG1 — DISMISSED** [review]

**Review by jerryshao — APPROVED** [review]

**Pull request merged**